lib/web/fetch/request.js

```diff
@@ -101,6 +103,8 @@ class Request {
       // 5. Set fallbackMode to "cors".
       fallbackMode = 'cors'
     } else {
+      this[kDispatcher] = input[kDispatcher]
```

Comment:
Shouldn't this like look like this? Allowing to override the dispatcher when creating a new request from an existing one.

```suggestion
      this[kDispatcher] = init.dispatcher || input[kDispatcher]
```

Reply:
Can you Provide a Pr and test?

Reply:
https://github.com/nodejs/undici/pull/2928